src/test/java/net/snowflake/ingest/streaming/internal/FlushServiceTest.java

```diff
@@ -605,8 +605,8 @@ public void testEncryptionDecryption()
         Base64.getEncoder().encodeToString("encryption_key".getBytes(StandardCharsets.UTF_8));
     String diversifier = "2021/08/10/blob.bdec";
 
-    byte[] encryptedData = Cryptor.encrypt(data, encryptionKey, diversifier);
-    byte[] decryptedData = Cryptor.decrypt(encryptedData, encryptionKey, diversifier);
+    byte[] encryptedData = Cryptor.encrypt(data, encryptionKey, diversifier, 0);
```

Comment:
could you add a IT test in `StreamingIngestIT` that covers multiple channels for multiple tables?

Reply:
I will try. I had a problem with Snowflake connection in devvm with tests.

Reply:
done